node/flatpak-node-generator.py

```diff
@@ -1713,6 +1725,8 @@ async def main() -> None:
     parser.add_argument('--nwjs-node-headers',
                         action='store_true',
                         help='Download the NW.js node headers')
+    parser.add_argument('--nwjs-ffmpeg', action='store_true',
+                        help='Download prebuilt ffmpeg for current NW.js version')
```

Comment:
Might be worth also editing the help string `--electron-ffmpeg` above, since it's now a bit confusing.

Reply:
Done.